### DISCUSSION TITLE
ar-cloud-anchor sample + azure

### DISCUSSION THREAD

**Discussion by MarekMZ** [sceneview/sceneview#504] [Q&A]
How to host spatial anchor on microsoft azure spatial anchor service?

**Comment by ThomasGorisse** [comment]
SceneView v4.0.0-rc.1 supports ARCore Cloud Anchors natively. For Azure Spatial Anchors, you'd need to integrate the Azure SDK separately.

For ARCore Cloud Anchors:
```kotlin
ARSceneView(
    sessionConfiguration = { session, config ->
        config.cloudAnchorMode = Config.CloudAnchorMode.ENABLED
    }
)
```

Azure Spatial Anchors is a separate SDK that works alongside ARCore. You can create anchors with SceneView's `ARSceneView`, then pass the underlying ARCore anchor to the Azure Spatial Anchors SDK for cloud persistence. The integration is at the ARCore level, not SceneView-specific.
